packages/edit-site/src/components/sidebar-navigation-screen-page/index.js

```diff
@@ -18,40 +34,238 @@ import SidebarNavigationScreen from '../sidebar-navigation-screen';
 import { unlock } from '../../private-apis';
 import { store as editSiteStore } from '../../store';
 import SidebarButton from '../sidebar-button';
+import SidebarNavigationSubtitle from '../sidebar-navigation-subtitle';
+import SidebarDetails from '../sidebar-navigation-data-list';
+import StatusLabel from './status-label';
+
+// Taken from packages/editor/src/components/time-to-read/index.js.
+const AVERAGE_READING_RATE = 189;
+
+function getPageDetails( page ) {
+	if ( ! page ) {
+		return [];
+	}
+	const details = [];
+
+	if ( page?.status ) {
+		details.push( {
+			label: 'Status',
```

Comment:
Should each of these `label` values be wrapper in translation (`__()`) calls?

Reply:
Yes! They should! 👍